tasks/watch.js

```diff
@@ -84,6 +84,20 @@ gulp.task('watch:images', () => {
 
 
 /**
+ * `watch:assets` Task
+ * -------------
+ * Runs the `copy:assets` task when there are changes in node modules.
+ *
+ */
+gulp.task('watch:assets', () => {
+
+    gulp.watch('node_modules/@justeat/**/*', ['copy:assets'])
```

Comment:
MIght be a good idea to make this path configurable in case we want to pull in assets from a different source in the future?